index.js

```diff
@@ -1,5 +1,4 @@
 module.exports = {
   cloaked_msg_id: require('./cloaked_msg_id/constants.json'),
-  derive_secret: require('./derive_secret/constants.json'),
-  tfk: require('./encoding/tfk.json')
```

Comment:
:fire:

I would like to make this a minor/ patch version change.
Given the spec exports things which are imported and used for computations in other modules this would currently make this a major breaking change

Propose:
If these are compatible, I think we should require and export the bfe.json here:

```js
const bfe = require('./encoding/bfe.json')

module.exports = {
  cloaked_msg_id: require('./cloaked_msg_id/constants.json'),
  derive_secret: require('./derive_secret/constants.json'),
  tfk: bfe,
  bfe
}
```

Reply:
Yes can do that